packages/block-library/src/categories/block.json

```diff
@@ -22,6 +22,10 @@
 		"showOnlyTopLevel": {
 			"type": "boolean",
 			"default": false
+		},
+		"showEmpty": {
+		  "type": "boolean",
```

Comment:
The indentation here needs to be updated(`tab`). There should be linting rule here, but it seems there were no complains 😄

Reply:
hello thanks for reviewing. It's fixed